app/controllers/admin/api/registry/policies_controller.rb

```diff
@@ -1,14 +1,17 @@
 # frozen_string_literal: true
 
 class Admin::Api::Registry::PoliciesController < Admin::Api::BaseController
+  self.access_token_scopes = :policy_registry
+
   clear_respond_to
   respond_to :json
 
   representer ::Policy
 
 
   # swagger
-  ##~ sapi = source2swagger.namespace("Account Management API")
+  ##~ sapi = source2swagger.namespace("Policy Registry API")
```

Comment:
I wonder if there is a need to separate Policy Registry from Account Management.
It is part of the account, like services

Reply:
@hallelujah Almost everything is part of account then 😄 Even Billing, but it has its own space and it is shown only if it is active, so I've followed the same philosophy of showing it only if it allowed to be used (because it has its own token and permission, and it is not allowed for master) 😄

Reply:
"Policies Registry API"

Reply:
I think it's actually correct in singular _Policy Registry..._